aion/web/elm/src/Panel/Validators.elm

```diff
@@ -31,6 +31,17 @@ subjectValidations =
     [ Forms.validateExistence ]
 
 
+categoryNameValidations : List Forms.FieldValidator
+categoryNameValidations =
+    let
+        categoryNameRegex =
+            "\\b([A-Z][a-z]+)\\b"
```

Comment:
```
^\s*(.+?)\s+$
```
You could do this for multi word categories. Did you check how your regex behaves when dealing with Unicode characters?